util/tracer_nvbit/tracer_tool/tracer_tool.cu

```diff
@@ -301,8 +305,11 @@ void nvbit_at_cuda_event(CUcontext ctx, int is_exit, nvbit_api_cuda_t cbid,
 
   if (first_call == true) {
     first_call = false;
-
-    if (mkdir("traces", S_IRWXU | S_IRWXG | S_IROTH | S_IXOTH) == -1) {
+    if (user_defined_folders == 1) {
```

Comment:
yeah, I would clean this all up.
just make the default cwd if the TRACE_FOLDER is ""